src/content/pages/volunteers.mdx

```diff
@@ -0,0 +1,120 @@
+
+This is a very people-first position and will require spending a considerable time at the registration desk talking to attendees, along with carrying out the following activities:
+
+* Handing out registration badges
+* Checking student IDs
+* Answering general questions.
+
+We want to call out that volunteers at the registration desk will generally not be able to attend talks for the duration of the task.
+
+## T-Shirt Handout
+
+* Handing out the conference T-shirt
+* This job involves standing and lifting light items.
+
+We want to call out that this job is located at the registration desk, and you will generally not be able to attend talks for the duration of the task.
+
+## Room Manager
+
+A room manager is responsible for ensuring everything in a room runs smoothly for the entirety of a slot. Some of the activities a room manager is expected to carry out involve:
+
+* Insuring people are seated before a talk, and the room is not overcrowded
+* Making sure the session chair and speakers are in the room in time
+* Keeping in touch with the AV team to ensure everything is in order
+* Keeping the Code of Conduct (CoC) in mind and contact CoC committee members if needed
+
+We want to call out that room managers might not be able to attend or focus on talks for the duration of the task.
+
+## Session Chairing
+
+A session chair plays a key role in introducing talks and ensuring a seamless transition between talks scheduled during a slot. The activities that a session chair is expected to carry out include:
+
+* Helping speakers in preparing their computers for the talks
+* Introducing the speakers to the audience
+* Signalling time left to the speakers
+* Making sure talks start and finish on time
+* Moderating Q&A session and help with passing around microphones and introduce remote attendees.
+
+## Set Up/Tear Down
+
+* Putting up signage in hallways for various provisions (Bathroom, Quiet Room, Childcare etc.)
+* Also on the last day (Friday afternoon for the main conference, Sunday for the sprints) to clear up the venue afterward
+* This job involves walking and carrying items, some of which maybe heavy.
+
+Runners and Ad-hoc Activities
+
+There are always some tasks which we simply cannot plan beforehand. Based on what's needed in the moment, you can be expected to do one or more of the following (not an exhaustive list):
+
+* Checking rooms to ensure all the necessary resources are in place
+* Helping out with Registration Desk duties, T-Shirt Handout, or Room Manager duties
+* Moving chairs, tables, or poster walls
+* Arranging transfers between venue and airport, etc. (Rarely)
+
+# How to sign up
+
+We are currently still working out the latest details for the volunteering program. (We are trying to get more perks enabled for your volunteering time).
+Until then, you can sign up for volunteering by sending an email to [volunteers@europython.eu](mailto:volunteers@europython.eu) .
+
+<div style={{textAlign: "center"}}>
```

Comment:
```suggestion
<div class="text-center">
```

Reply:
I copied from visa. But looks somehow better to me.